backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -92,7 +94,32 @@ public PxpVerifyResponse getPatientLinkVerify(
     return new PxpVerifyResponse(p, os, te);
   }
 
+  /**
+   * Given a patient link ID, returns the name of the patient for whom the test was conducted, in
+   * lightly-obfuscated form (e.g. "John Doe" -> "John D").
+   *
+   * <p>This endpoint is unauthorized and therefore does not return fully identifying data.
+   *
+   * @param request
+   * @return an obfuscated patient name
+   */
+  @GetMapping("/patient-name")
```

Comment:
I dont like how this is not checking if the patient link is not expired, we can actually use this to just show the link expired right away without the need to put dob to check for that

Reply:
That's a good idea, added in 2bcf681